e2e/utils.ts

```diff
@@ -164,16 +174,22 @@ export const prepareStandaloneSetup = (fixtureName: string) => {
         cwd: wakuDir,
         stdio: 'inherit',
       });
+      const wakuPackageTgz = join(standaloneDir, `waku-${version}.tgz`);
+      const installScript = PACKAGE_INSTALL[packageManager](wakuPackageTgz);
+      execSync(installScript, { cwd: standaloneDir, stdio: 'inherit' });
       execSync(
         `npm install --force ${join(standaloneDir, `waku-${version}.tgz`)}`,
         { cwd: standaloneDir, stdio: 'inherit' },
       );
     }
     if (mode !== 'DEV' && !built) {
-      rmSync(`${standaloneDir}/dist`, { recursive: true, force: true });
+      rmSync(`${standaloneDir}${packageDir}/dist`, {
+        recursive: true,
+        force: true,
+      });
       execSync(
         `node ${join(standaloneDir, './node_modules/waku/dist/cli.js')} build`,
-        { cwd: standaloneDir },
+        { cwd: join(standaloneDir, packageDir) },
```

Comment:
Make it consistent with L186. Either use `join` for both, or use string concatenation for both. L209 too.

Reply:
Yeah, thanks. Updated L186 to `join`.